storage/azblob/options.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package azblob
+
+const (
+	BufferSizeMin     = 4 * 1024
+	BufferSizeDefault = 4 * BufferSizeMin
+)
+
+type Options struct {
+	ConnectionString *string
+	SharedKey        *SharedKeyCredentials
+
+	BufferSize int
+
+	ContentType    *string
+	FilenameSuffix *string
+}
+
+func NewOptions(opts ...*Options) *Options {
+	opt := &Options{
+		BufferSize: 4 * BufferSizeMin,
```

Comment:
could we get a comment in the code on why it is 4 times the size? (just in passing `BufferSizeBytes` perhaps?)